src/components/modules/document/DocSingleCreation/modules/UploadStatus/UploadStatus.interactor.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <upload-status-layout
+    :is-pending="isPending"
+    :is-verified="isVerified"
+    @continue="onContinue"
+    @create="onCreate"
+  />
+</template>
+
+<script>
+import UploadStatusLayout from '@/components/modules/document/common/UploadStatusLayout';
```

Comment:
Why you are using `Layout` here, not on `view` layer?

Reply:
what will be in `view` layer?